packages/components/src/index.ts

```diff
@@ -1026,6 +1028,8 @@ export {
     SAMPLE_DATA_EXPORT_CONFIG,
     SAMPLE_EXPORT_CONFIG,
     SAMPLE_INSERT_EXTRA_COLUMNS,
+    SAMPLE_DOMAIN_DEFAULT_SYSTEM_FIELDS,
+    SAMPLE_DOMAIN_INVENTORY_SYSTEM_FIELDS,
```

Comment:
minor: what are your thoughts on exposing a helper function instead of these two constants? a function like `getSampleDomainDefaultSystemFields(moduleContext)` could then make the check for `isFreezerManagementEnabled` itself and then return accordingly. This would remove a bit of the duplication in the LKS and app cases.

Reply:
I like this! Nice to export one fn rather than two constants, too. Update made.